zorch/commit/testing/ajtai_test.py

```
@@ -0,0 +1,190 @@
+        self.assertTrue(self.scheme.verify(self.matrix, commitment, witness))
+
+    def test_the_commitment_is_additively_homomorphic(self) -> None:
+        """com(s1) + com(s2) == com(s1 + s2) — the folding prerequisite."""
+        s1 = _ternary_witness(self.ring, self.rng, _COLS)
+        s2 = _ternary_witness(self.ring, self.rng, _COLS)
+        lhs = self.ring.add(
+            self.scheme.commit(self.matrix, self.ring.ntt(s1)),
+            self.scheme.commit(self.matrix, self.ring.ntt(s2)),
+        )
+        rhs = self.scheme.commit(self.matrix, self.ring.ntt(self.ring.add(s1, s2)))
+        for got, want in zip(lhs.limbs, rhs.limbs):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)